modules/cli/src/main/scala/scala/cli/commands/config/Config.scala

```diff
@@ -298,4 +300,32 @@ object Config extends ScalaCommand[ConfigOptions] {
       }
     }
   }
+
+  /** If the new value is the same as the previous value, ask user for confirmation or suggest to
```

Comment:
Something's mixed up in this scaladoc - if values are the same then no confirmation nor --force flag is needed, right?

Reply:
Right! Good catch.